examples/samples/movingMessagesToDLQ.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  delay,
+  generateUuid,
+  SendableMessageInfo,
+  OnMessage,
+  OnError,
+  MessagingError,
+  ServiceBusMessage,
+  ReceiveMode,
+  Namespace
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const queuePath = process.env.QUEUE_NAME || "";
+const deadLetterQueuePath = Namespace.getDeadLetterQueuePathForQueue(queuePath);
+const receiveClientTimeoutInMilliseconds = 10000;
+console.log("str: ", str);
+console.log("queue path: ", queuePath);
+console.log("deadletter queue path: ", deadLetterQueuePath);
+
+let ns: Namespace;
+
+/*
+    This sample demonstrates scenarios as to how messages can be explicitly moved to the DLQ.
+    For other implicit ways when messages get moved to DLQ, refer to -
+    https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dead-letter-queues
+
+    CAUTION: Running this sample may cause all messages in main queue to be moved to DLQ.
```

Comment:
Users who have a non empty queue and those who might have skipped this part will end up having all their messages moved to the dead letter queue.

We can reduce the risks by using `receiveBatch(1)` and deadlettering just that 1 message.
The risk then would be if the queue was not empty, then there is a chance that instead of the message we explicitly sent to the queue (Creamy Chicken Pasta), some other message gets deadlettered. But this anyway can happen with the current code.

Reply:
Another option is to keep the current code, and just add a check on the message body and dead letter the one that we want and ignore the rest?

Reply:
Updated code to use the receiveBatch(1) idea we discussed about.